README.md

```diff
@@ -3,4 +3,10 @@
 [![Build Status](https://travis-ci.org/pmorie/go-open-service-broker-skeleton.svg?branch=master)](https://travis-ci.org/pmorie/go-open-service-broker-skeleton "Travis")
 
 A go quickstart for creating service brokers that implement the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
\ No newline at end of file
+[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
+Broker authors implement an interface that uses the same types as the
+[go-open-service-broker-client](https://github.com/pmorie/go-open-service-broker-client)
+project.
+
+## Who should use this project?
+
```

Comment:
Oops, that was for another branch. Should not pack for a week and try to code at same time